frontend/archive/src/main/java/org/pytorch/serve/archive/model/ModelArchive.java

```diff
@@ -25,12 +26,14 @@ public class ModelArchive {
     private String url;
     private File modelDir;
     private boolean extracted;
+    private ModelConfig modelConfig;
 
     public ModelArchive(Manifest manifest, String url, File modelDir, boolean extracted) {
         this.manifest = manifest;
         this.url = url;
         this.modelDir = modelDir;
         this.extracted = extracted;
+        this.modelConfig = null;
```

Comment:
hmm we might need to rename things cause we already have frontend model config in config.properties - how do you feel about handler config?